### PR TITLE
Add known issues around transit managed keys

### PR DESCRIPTION
**Pull request by stevendpclark**
- Document known issue around managed key encryption failure with Cloud KMS backed keys and the failure to sign with managed keys

### CODE REVIEW

File: website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#### Affected versions
+
+- 1.14.0+ up to 1.14.3 inclusively.
+
+#### Issue
+
+The Transit sign API call responds with an error "requested version for signing does not contain a private part" when
+a Transit sign request uses a managed key.
+
+<Note>Signing APIs are not affected with non-managed key, key types</Note>
```
Comment (cipherboy) [inline website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx:12]:
```suggestion
<Note>Signing APIs are not affected with transit native keys</Note>
```

perhaps?

Comment (cipherboy) [reply website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx:12]: Just trying to differentiate the two clearly :-)

Comment (stevendpclark) [reply website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx]: Good suggestion, thanks!

File: website/content/partials/known-issues/transit-managed-keys-panics.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#### Affected versions
+
+- 1.13.1+ up to 1.13.7 inclusively
+- 1.14.0+ up to 1.14.3 inclusively.
+
+#### Issue
+
+Vault will panic when a Transit encryption API call is backed by a Cloud KMS managed key (Azure, GCP, AWS).
+
+<Note>Encryption and decryption APIs are not affected with PKCS#11 managed keys and non-managed key, key types</Note>
```
Comment (cipherboy) [inline website/content/partials/known-issues/transit-managed-keys-panics.mdx:12]:
```suggestion
<Note>Encryption and decryption APIs are not affected with PKCS#11 managed keys and transit native keys</Note>
```

Comment (schavis) [inline website/content/partials/known-issues/transit-managed-keys-panics.mdx]:
```suggestion
<Note>

The issue does not affect encryption and decryption with the following key types:

- PKCS#11 managed keys
- non-managed keys

</Note>
```

Style correction: make important information easy to scan and parse

File: website/content/partials/known-issues/transit-managed-keys-panics.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+### Transit Encryption with Cloud KMS managed keys causes a panic
+
+#### Affected versions
+
+- 1.13.1+ up to 1.13.7 inclusively
+- 1.14.0+ up to 1.14.3 inclusively.
+
+#### Issue
+
+Vault will panic when a Transit encryption API call is backed by a Cloud KMS managed key (Azure, GCP, AWS).
```
Comment (schavis) [inline website/content/partials/known-issues/transit-managed-keys-panics.mdx:10]:
```suggestion
Vault panics when it receives a Transit encryption API call that is backed by a Cloud KMS managed key (Azure, GCP, AWS).
```

Style correction: write in active voice, make it clear who/what is taking the action

File: website/content/partials/known-issues/transit-managed-keys-panics.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+### Transit Encryption with Cloud KMS managed keys causes a panic
+
+#### Affected versions
+
+- 1.13.1+ up to 1.13.7 inclusively
+- 1.14.0+ up to 1.14.3 inclusively.
```
Comment (schavis) [inline website/content/partials/known-issues/transit-managed-keys-panics.mdx:6]:
```suggestion
- 1.14.0+ up to 1.14.3 inclusively
```

Style correction: format all line items consistently

File: website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+### Transit Sign API calls with managed keys fail
+
+#### Affected versions
+
+- 1.14.0+ up to 1.14.3 inclusively.
```
Comment (schavis) [inline website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx:5]:
```suggestion
- 1.14.0+ up to 1.14.3 inclusively
```

Style correction: format similar page elements consistently

File: website/content/partials/known-issues/transit-managed-keys-panics.mdx (hunk 2 above)
Comment (stevendpclark) [reply website/content/partials/known-issues/transit-managed-keys-panics.mdx]: Thanks you beat me to updating from cli.

Comment (cipherboy) [reply website/content/partials/known-issues/transit-managed-keys-panics.mdx]: Whoops :P

**Review by cipherboy — APPROVED** [review]

File: website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+#### Issue
+
+The Transit sign API call responds with an error "requested version for signing does not contain a private part" when
+a Transit sign request uses a managed key.
```
Comment (schavis) [inline website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx:10]:
```suggestion
Vault responds to Transit sign API calls with the following error when the request uses a managed key:

`requested version for signing does not contain a private part`
```

Style correction: Make key information easy to scan for, make it clear who/what is taking action

File: website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx (hunk 1 above)
Comment (schavis) [inline website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx]:
```suggestion
<Note>

The issue does not affect signing with the following key types:

- non-managed keys

</Note>
```

Style correction: make key information easy to scan

**Review by schavis — CHANGES_REQUESTED** [review]

File: website/content/partials/known-issues/transit-managed-keys-panics.mdx (hunk 2 above)
Comment (stevendpclark) [reply website/content/partials/known-issues/transit-managed-keys-panics.mdx]: Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b, just tweaked non-managed keys to Transit native keys

File: website/content/partials/known-issues/transit-managed-keys-panics.mdx (hunk 3 above)
Comment (stevendpclark) [reply website/content/partials/known-issues/transit-managed-keys-panics.mdx]: Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b

File: website/content/partials/known-issues/transit-managed-keys-panics.mdx (hunk 4 above)
Comment (stevendpclark) [reply website/content/partials/known-issues/transit-managed-keys-panics.mdx]: Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b

File: website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx (hunk 5 above)
Comment (stevendpclark) [reply website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx]: Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b

File: website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx (hunk 1 above)
Comment (stevendpclark) [reply website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx]: Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b, just tweaked non-managed keys to Transit native keys

File: website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx (hunk 6 above)
Comment (stevendpclark) [reply website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx]: Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b and updated missing newline within https://github.com/hashicorp/vault/pull/23080/commits/8672872bc96e136c707c01da8d32cc08e0f97c2f

**Review by schavis — APPROVED** [review]

**Comment by stevendpclark** [comment]
Thanks all!

**Pull request merged**